ftw/simplelayout/browser/resources/integration.theme.scss

```diff
@@ -35,6 +36,25 @@
   }
 }
 
+  .icons-on .sl-image .colorboxLink{
+    @extend .fa-expand;
+    @extend .fa-icon;
+    &:before {
+    color: white;
```

Comment:
@Tschanzt can you fix the indenting?
Please use the same indenting settings as in the rest of the document (local consistency) 😉 
thanks!